candi/version_map.yml

```diff
@@ -174,7 +188,6 @@ k8s:
       # kubeControllerManager: sha256 digest isn't needed because this component is compiled from source
       kubeScheduler: sha256:415afb38d7d610177753f1bd62fd6ef1bf3eec85cc17e100735775d7b7de1ccf
       # kubeProxy: sha256 digest isn't needed because this component is compiled from source for that version of kubernetes
```

Comment:
```suggestion
      # kubeProxy: sha256 digest isn't needed for this version of kubernetes because this component is compiled as a module image with a special patch
```